.github/workflows/govulncheck.yml

```diff
@@ -0,0 +1,17 @@
+name: govulncheck
+on:
+  pull_request:
+
+jobs:
+  govulncheck:
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest, windows-latest ]
+    name: vulncheck
+    runs-on: ${{  matrix.os }}
+    steps:
+      - id: govulncheck
+        uses: golang/govulncheck-action@v1
+        with:
+           go-version-input: 1.20.6
```

Comment:
Shouldn't this be in sync with what's in https://github.com/elastic/elastic-agent/blob/main/.go-version (currently `1.20.5`)?

Reply:
Yes it should, but https://github.com/elastic/elastic-agent/pull/3087 already exists and will probably merge today.